### PR TITLE
PMM-9662 Platform: Add GetContactInformation API

### PR DESCRIPTION
**Pull request by palash25**
PMM-9662

Build: SUBMODULES-2367

- [ ] Links to other linked pull requests (optional).

### CODE REVIEW

File: services/platform/platform.go [hunk 1]
```diff
@@ -537,3 +538,80 @@ func convertEntitlement(ent *entitlementResponse) (*platformpb.OrganizationEntit
...
+		NewTicketURL string `json:"new_ticket_url"` //nolint:tagliatelle
+	} `json:"contact"`
+}
+
+// GetContactInformation fetches contact information of the Customer Success employee assigned to the Percona customer from Percona Portal.
+func (s *Service) GetContactInformation(ctx context.Context, req *platformpb.GetContactInformationRequest) (*platformpb.GetContactInformationResponse, error) {
+	userAccessToken, err := s.grafanaClient.GetCurrentUserAccessToken(ctx)
+	if err != nil {
+		if errors.Is(err, grafana.ErrFailedToGetToken) {
+			return nil, status.Error(codes.Unauthenticated, "Failed to get access token. Please sign in using your Percona Account.")
```
Comment (oksana-grishchenko) [inline services/platform/platform.go:557]: should an error log also be written here before return?

File: services/platform/platform.go [hunk 2]
```diff
@@ -537,3 +538,80 @@ func convertEntitlement(ent *entitlementResponse) (*platformpb.OrganizationEntit
...
+		CustomerSuccess: &platformpb.GetContactInformationResponse_CustomerSuccess{
+			Name:  platformResp.Contact.CustomerSuccess.Name,
+			Email: platformResp.Contact.CustomerSuccess.Email,
+		},
+		NewTicketUrl: platformResp.Contact.NewTicketURL,
+	}
+
+	// Platform account is not linked to ServiceNow.
+	if res.CustomerSuccess.Email == "" {
+		return nil, nonCustomerAccount
```
Comment (oksana-grishchenko) [inline services/platform/platform.go:613]: Here also - should an error log also be here before return?

**Review by oksana-grishchenko — APPROVED** [review]
About the failing CI - Artem posted the other day https://perconacorp.slack.com/archives/GJAGH01PD/p1646417694239499 about this particular problem

File: api-tests/server/platform_test.go [hunk 3]
```diff
@@ -209,4 +209,34 @@ func TestPlatform(t *testing.T) {
...
+
+			// Confirm we are connected to Portal.
+			settings, err := serverClient.GetSettings(nil)
+			require.NoError(t, err)
+			require.NotNil(t, settings)
+			assert.True(t, settings.GetPayload().Settings.ConnectedToPlatform)
+
+			resp, err := client.GetContactInformation(&platform.GetContactInformationParams{Context: pmmapitests.Context})
+			require.NoError(t, err)
+			require.NotEmpty(t, resp.GetPayload().CustomerSuccess.Email)
```
Comment (BupycHuk) [inline api-tests/server/platform_test.go:237]: should we check if email is equal to email used to connect?

**Review by BupycHuk — APPROVED** [review]

File: api-tests/server/platform_test.go (hunk 3 above)
Comment (palash25) [reply api-tests/server/platform_test.go:237]: no this is not the same as the user's email. this is the email of the customer success employee that deals with the company that the percona platform user belongs to

Comment (BupycHuk) [reply api-tests/server/platform_test.go:237]: oh, ok, thanks for explanation

**Pull request merged**